test-suites/test-aave/aportoken.spec.ts

```diff
@@ -0,0 +1,240 @@
+        amountToDeposit.add(balanceBefore)
+      );
+    });
+
+    it('should mint successfully when both feed and heartbeat are set', async () => {
+      // Make sure feed and heartbeat values are what we're testing for
+      await aWbtc._setFeed(mockV3Aggregator.address);
+      expect(await aWbtc.feed()).to.equal(mockV3Aggregator.address);
+      await aWbtc._setHeartbeat(ONE_DAY_SECONDS);
+      expect(await aWbtc.heartbeat()).to.equal(ONE_DAY_SECONDS);
+
+      // Deposit WBTC - the aToken will call the feed before minting to check PoR
+      const balanceBefore = await aWbtc.balanceOf(regularUser.address);
+      await pool
+        .connect(regularUser.signer)
+        .deposit(wbtc.address, amountToDeposit, regularUser.address, '0');
+      expect(await aWbtc.balanceOf(regularUser.address)).to.equal(
+        balanceBefore.add(amountToDeposit)
+      );
+    });
+
+    it('should mint successfully when feed decimals < underlying decimals', async () => {
+      // Re-deploy aggregator with fewer decimals
+      const currentWbtcSupply = await wbtc.totalSupply();
+      mockV3Aggregator = await deployMockV3Aggregator(
+        '6',
+        currentWbtcSupply.div(exp(1, 2)).toString()
+      );
+
+      // Make sure feed and heartbeat values are set
+      await aWbtc._setFeed(mockV3Aggregator.address);
+      expect(await aWbtc.feed()).to.equal(mockV3Aggregator.address);
+      await aWbtc._setHeartbeat(ONE_DAY_SECONDS);
+      expect(await aWbtc.heartbeat()).to.equal(ONE_DAY_SECONDS);
+
+      // Deposit WBTC - the aToken will call the feed before minting to check PoR
+      const balanceBefore = await aWbtc.balanceOf(regularUser.address);
+      await pool
+        .connect(regularUser.signer)
+        .deposit(wbtc.address, amountToDeposit, regularUser.address, '0');
+      expect(await aWbtc.balanceOf(regularUser.address)).to.equal(
+        balanceBefore.add(amountToDeposit)
+      );
+    });
+
+    it('should mint successfully when feed decimals > underlying decimals', async () => {
+      // Re-deploy aggregator with more decimals
+      const currentWbtcSupply = await wbtc.totalSupply();
+      mockV3Aggregator = await deployMockV3Aggregator(
+        '18',
+        currentWbtcSupply.mul(exp(1, 10)).toString()
+      );
+
+      // Make sure feed and heartbeat values are set
+      await aWbtc._setFeed(mockV3Aggregator.address);
+      expect(await aWbtc.feed()).to.equal(mockV3Aggregator.address);
+      await aWbtc._setHeartbeat(ONE_DAY_SECONDS);
+      expect(await aWbtc.heartbeat()).to.equal(ONE_DAY_SECONDS);
+
+      // Deposit WBTC - the aToken will call the feed before minting to check PoR
+      const balanceBefore = await aWbtc.balanceOf(regularUser.address);
+      await pool
+        .connect(regularUser.signer)
+        .deposit(wbtc.address, amountToDeposit, regularUser.address, '0');
+      expect(await aWbtc.balanceOf(regularUser.address)).to.equal(
+        balanceBefore.add(amountToDeposit)
+      );
+    });
+
```

Comment:
If you could also add a test for `'should mint successfully when underlying supply == proof-of-reserves'`. This will help with @rbrogan-git's comment.

Reply:
Added 🚀